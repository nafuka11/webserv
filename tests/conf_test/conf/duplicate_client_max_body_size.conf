server {
    client_max_body_size 100;
    client_max_body_size 1000;
}