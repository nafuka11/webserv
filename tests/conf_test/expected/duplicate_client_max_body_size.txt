webserv: duplicate "client_max_body_size" directive in ./conf/duplicate_client_max_body_size.conf:3
